docs/conceptual/runtime/frame.md

```diff
@@ -93,191 +93,191 @@ authorities.
 
-The Offences module tracks reported offences.
+The Offences pallet tracks reported offences.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_offences/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/offences/src/lib.rs)
 
 ### Randomness Collective Flip
 
-The Randomness Collective Flip module provides a `random` function that generates low-influence
+The Randomness Collective Flip pallet provides a `random` function that generates low-influence
 random values based on the block hashes from the previous `81` blocks.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_randomness_collective_flip/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/randomness-collective-flip/src/lib.rs)
 
 ### Scored Pool
 
-The Scored Pool module maintains a scored membership pool where the highest scoring entities are
+The Scored Pool pallet maintains a scored membership pool where the highest scoring entities are
 made members.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_scored_pool/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/scored-pool/src/lib.rs)
 
 ### Session
 
-The Session module allows validators to manage their session keys, provides a function for changing
+The Session pallet allows validators to manage their session keys, provides a function for changing
 the session length, and handles session rotation.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_session/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/session/src/lib.rs)
 
 ### Staking
 
-The Staking module is used to manage funds at stake by network maintainers.
+The Staking pallet is used to manage funds at stake by network maintainers.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_staking/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/staking/src/lib.rs)
 
 ### Sudo
 
-The Sudo module allows for a single account (called the "sudo key") to execute dispatchable
+The Sudo pallet allows for a single account (called the "sudo key") to execute dispatchable
 functions that require a `Root` origin or designate a new account to replace them as the sudo key.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_sudo/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/sudo/src/lib.rs)
 
 ### Timestamp
 
-The Timestamp module provides functionality to get and set the on-chain time.
+The Timestamp pallet provides functionality to get and set the on-chain time.
 
 * [Docs](https://substrate.dev/rustdocs/master/pallet_timestamp/index.html)
 * [Source](https://github.com/paritytech/substrate/blob/master/frame/timestamp/src/lib.rs)
 
 ### Transaction Payment
 
-The Transaction Payment module provides the basic logic needed to pay the absolute minimum amount
+The Transaction Payment pallet provides the basic logic needed to pay the absolute minimum amount
```

Comment:
Not exactly true, it still lets you add tips. I would say,

The Transaction Payment palled provides the logic to compute pre-dispatch transaction fees.